src/language-css/parser-postcss.js

```diff
@@ -40,8 +40,17 @@ function parseValueNodes(nodes) {
       node.group.groups[0].groups[0].value === "data" &&
       node.group.groups[0].groups[1].type === "colon" &&
       node.group.groups[0].groups[1].value === ":";
+    const isUnquotedURLWithNumber =
+      node.type === "func" &&
+      node.value === "url" &&
+      node.group &&
+      node.group.groups &&
+      node.group.groups[0] &&
+      node.group.groups[0].groups &&
+      node.group.groups[0].groups.length > 2 &&
+      node.group.groups[0].groups.find(group => group.type === "number");
```

Comment:
I think better implement utils `isValidURL` and print all valid url in one line, it is solve many problem, not only with numbers, anyway good work!

Reply:
Good point. Could you please figure out any other cases?

Reply:
@evilebottnawi You mean extract `isUnquotedDataURLCall` and `isUnquotedURLWithNumber` to `isValidURL ` util? Maybe I haven't get your point.

Reply:
@jasonxia23 i thinks we need get logic from here https://nodejs.org/docs/latest/api/url.html

> A TypeError will be thrown if the input or base are not valid URLs. Note that an effort will be made to coerce the given values into strings. For instance:

```js
const myURL = new URL({ toString: () => 'https://example.org/' });
// https://example.org/
```

And it is valid url we stringify group

Reply:
Well, I think CSS `<url>` is not same as Javascript `URL`.

For example, I can use `url(foo.png)` but `foo.png` is not "VALID" for `URL`

Reply:
Technically, at the parsing level the stuff inside `url()` does not have to be a valid URL: https://www.w3.org/TR/css-syntax-3/#url-unquoted-diagram

Reply:
@jasonxia23 let's use logic for check what @lydell write above, it is good solution

Reply:
@evilebottnawi Maybe I should turn to `postcss-values-parser`. Prettier uses `postcss-values-parser` other than `postcss-value-parser`.